scripts/spectra_convert/convert_to_fits_function.py

```diff
@@ -0,0 +1,61 @@
+import os
+import sys
+import logging
+from urllib.parse import unquote
+import astropy.io.fits as fits
+from astropy.table import Table
+from matplotlib import pyplot as plt
+from specutils import Spectrum1D
+from header_function import *
+logger = logging.getLogger('SIMPLE')
+import astropy.units as u
+
+
+# Logger setup
+logger.propagate = False  # prevents duplicated logging messages
+LOGFORMAT = logging.Formatter('%(asctime)s %(levelname)s: %(message)s', datefmt='%m/%d/%Y %I:%M:%S%p')
+ch = logging.StreamHandler(stream=sys.stdout)
+ch.setFormatter(LOGFORMAT)
+# To prevent duplicate handlers, only add if they haven't been set previously
+if not len(logger.handlers):
+    logger.addHandler(ch)
+logger.setLevel(logging.INFO)
+
+
+def convert_to_fits(spectrum_info_all, spectrum_table, header):
+    # TODO: add docstring with expected keywords
+    # TODO: add error handling for expected keywords
+
+    object_name =header['OBJECT']
+
+ # gives original name of file
+   # spectrum_info_all['history'] = spectrum_info_all['generated_history']  # shows where file came from
+
```

Comment:
add another 'history' card here stating that the spectrum was made with the SIMPLE convert_to_fits.py function.

`header['HISTORY'] = "File made with the SIMPLE convert_to_fits.py function"`

Make sure this ADDS an additional history card and doesn't overwrite.